spec/models/classification_spec.rb

```diff
@@ -539,6 +539,29 @@
     end
   end
 
+  describe "name2tag" do
+    let(:root_ns)   { "/managed" }
```

Comment:
Looking at name2tag implementation, it has (imho surprising) behavior of preferring given namespace (defaulting to `/managed`) over the namespace of the parent.
Consider adding explicit test for that e.g. `name2tag("test_entry", parent, "/other_ns")`.

Reply:
@cben This close enough - or you have other ideas? https://github.com/ManageIQ/manageiq/pull/18305/files#diff-5377fe1d33c36f7bdd9b5a7cf5937383R542